client/src/components/IndexPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+const IndexPage = props => {
+    const [trail, setTrail] = useState([])
+
+    const getTrails = async () => {
+        try {
+            const response = await fetch("/api/v1/index")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            setTrail(body.trail)
```

Comment:
since it's a collection of `trips` this variable should be plural instead of `body.trip`